gulpfile.js

```diff
@@ -18,37 +19,56 @@ var paths = {
     scss: './sass/*.scss'
 };
 
+/* HTML task */
+gulp.task('html', function () {
+    gulp.src('src/*.html')
+        .pipe(gulp.dest('build'));
+});
+
+/* Images task */
+gulp.task('images', () =>
+    gulp.src('src/img/*')
+        .pipe(gulp.dest('build/img'))
+);
+
+/* Fonts task */
+gulp.task('fonts', () =>
```

Comment:
Icomoon font too should be available as npm

Reply:
This should be called 'icons' rather 'images', but I didn't rename it. Thing is this folder contains a custom set of icons which are used in the project and they can't be added via npm.